symfony/plugins/orangehrmLeavePlugin/Service/LeaveRequestService.php

```diff
@@ -510,14 +510,15 @@ public function getLeaveStatusesByNames(array $names): array
 
     /**
      * @param Leave[] $leaves
+     * @param Leave[] $allLeavesOfLeaveRequest
      * @return DetailedLeave[]
      */
-    public function getDetailedLeaves(array $leaves): array
+    public function getDetailedLeaves(array $leaves, array $allLeavesOfLeaveRequest): array
```

Comment:
@KaveeshaSilva1996 make sure to find usage before refactoring, since this method used in `symfony/plugins/orangehrmLeavePlugin/Api/BulkLeaveAPI.php` also